Automattic/newspack-theme
# Fix: correct pullquote font size for WP 6.1

### All Submissions:


### Changes proposed in this Pull Request:

In WordPress 6.1, the `.wp-block-pullquote` class gets a font size, which scales up the block in the Newspack theme. 

This PR prevents the block from looking different with this update. 

Closes #1956.

### How to test the changes in this Pull Request:

1. Start with the Newspack theme, and add a few pullquote blocks to the content -- make sure to align a couple left or right.
2. Update your test site to the latest WP 6.1 release candidate (you can use the [WordPress Beta Tester](https://wordpress.org/plugins/wordpress-beta-tester/) plugin to do this).
3. View the pullquote block on the front end and notice it's scaled up; it should appear the same in the editor though:

<img width="843" alt="image" src="https://user-images.githubusercontent.com/177561/197407241-88682899-df63-4075-8bfd-7c79efa5b293.png">

4. Apply this PR and run `npm run build`
5. Confirm that the pullquote block displays at the correct size again (on desktop, 28px for the pullquote text and 14px for the citation; on mobile, 25.2px for the pullquote text and 12.6px for the citation. If the block is aligned left or right, on desktop it should be 24px for the pullquote text and 14px for the citation; on mobile it should be 18px for the text and 12.6px for the citation).

<img width="812" alt="image" src="https://user-images.githubusercontent.com/177561/197407263-d52692a8-6bc1-4a98-aac2-141e3e50e67e.png">

6. Most of the child themes -- Nelson, Sacha, Katharine and Scott -- scale up the pullquote block to the same, slightly larger font size. Switch to one of these themes and confirm that the font size is still correct (the font size only changes on desktop -- it should be 36px for the body of the pullquote, or 28px when the block is aligned left or right.
7. Downgrade back to WP 6.0.3 and confirm this change doesn't affect anything.

### Other information:
